pinot-common/src/test/java/com/linkedin/pinot/common/segment/fetcher/SegmentFetcherFactoryTest.java

```diff
@@ -47,7 +48,7 @@ public void testInitSegmentFetcherFactory() throws Exception {
     SegmentFetcherFactory.getPreloadSegmentFetchers().put("https", mockHttpsFetcher);
     SegmentFetcherFactory.getPreloadSegmentFetchers().put(replacableProto, replacableFetcher);
 
-    SegmentFetcherFactory.initSegmentFetcherFactory(conf);
+    SegmentFetcherFactory.initSegmentFetcherFactory(conf.subset(CommonConstants.Server.PREFIX_OF_CONFIG_OF_SEGMENT_FETCHER_FACTORY));
```

Comment:
We would be fine to change the config to the latest key. However, I am not sure if any of other users might be relying on this configuration key. If you or other contributors are aware of other users using this configuration we should think about supporting backward compatibility.

Reply:
Yes, there is a chance that other users may be using pinot.server config in the controller between Sep and now. We will take that chance since this is a relatively new feature and small time window. The worst that will happen is that the hdfs fetcher (or any other custom fetchers that they may have added) will fail to initialize, and they will need to re-configure. Keeping pinot.server config inside controller is bad for the long run. Unfortunate that it slipped through the code.